.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Build compliance-masonry
+        run: go build cmd/compliance-masonry/compliance-masonry.go
+
+      - name: Build masonry
+        run: go build cmd/masonry/masonry.go
+
+      - name: Run tests
+        run: |
+          go test $(go list ./...) -covermode=count -coverprofile=coverage.txt
+          go tool cover -func coverage.txt
```

Comment:
I might be completely full of it, but I think we can get rid of doing the `go list ./...` command, and replace both of these with...

```sh
go test -coverprofile=coverage.txt -covermode=count ./...
```

I think, at least. That's based on what we experimented with in (https://github.com/opencontrol/compliance-masonry/pull/355#issuecomment-681483319) and linked to this `go` issue (https://github.com/golang/go/issues/19090). not a huge deal either way, just putting it out there for convenience